go-controller/pkg/ovn/egressfirewall_test.go

```diff
@@ -608,6 +615,257 @@ var _ = ginkgo.Describe("OVN EgressFirewall Operations", func() {
 				gomega.Expect(err).NotTo(gomega.HaveOccurred())
 
 			})
+			ginkgo.It("egress firewall with node selector updates during node update", func() {
```

Comment:
we either should add `gwMode` in the name here, or if you don't need to test that for gateway modes, move it outside of `for _, gwMode := range []config.GatewayMode` loop
same for the second test

Reply:
I think this was just something I missed when I rebased. I think in the past the tests were split into local and shared and for create vs update and it didnt make sense. iirc you fixed it. I'll update the test to use the right gw mode.